apis/apiextensions/fn/proto/v1beta1/run_function.pb.html

```diff
@@ -0,0 +1,509 @@
+---
+title: apiextensions.fn.proto.v1beta1
+layout: protoc-gen-docs
+generator: protoc-gen-docs
+number_of_entries: 10
+---
```

Comment:
This front matter makes the HTML file usable as Hugo content.

Allegedly you're supposed to be able to influence it with comments in the proto file, but I can't seem to get it to work.

https://pkg.go.dev/istio.io/tools/cmd/protoc-gen-docs#readme-front-matter